port/zos390/omrosdump.c

```diff
@@ -321,7 +326,7 @@ tdump_wrapper(struct OMRPortLibrary *portLibrary, char *filename, char *dsnName)
 static intptr_t
 tdump(struct OMRPortLibrary *portLibrary, char *asciiLabel, char *ebcdicLabel, uint32_t *returnCode, uint32_t *reasonCode)
 {
-	struct ioparms_t {
+	struct ioparmss_t {
```

Comment:
Did you mean to add an `s` here (that's not present on line 342)?

Reply:
Yes, this was unintentional, have reverted this.